test/StandardAssetRegistry.js

```diff
@@ -242,6 +242,56 @@ contract('StandardAssetRegistry', accounts => {
     })
   })
 
+  describe('Transacions Related', () => {
+    it('ensures the assets is pass to another owner after transfer', async () => {
+      const CONTENT_DATA = 'ramdom data'
+      const firstOwner = accounts[accounts.length - 2]
+      await registry.generate(3, CONTENT_DATA, { from: creator })
+      await registry.transfer(firstOwner, 3)
+      const newOwner = await registry.holderOf(3)
+      newOwner.should.be.equal(firstOwner)
+    })
+    it('ensures the sender owns the asset that is trying to send', async () => {
+      const CONTENT_DATA = 'ramdom data'
+      const randomOwner = accounts[accounts.length - 2]
+      const buyer = accounts[accounts.length - 3]
+      await registry.generate(4, CONTENT_DATA, { from: randomOwner })
+      await assertRevert( registry.transfer(buyer, 4) )
+    })
+    it('ensure only the first operation owns the asset when multiple are exec', async () => {
```

Comment:
clarify: `after receiving an asset, the receiver can't transfer the sender's other assets`